test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -176,4 +185,35 @@ private String executeAndRead(String command, Object... args)
             throw new RuntimeException(e);
         }
     }
+
+    public void preserveClusterLogs(String testName)
```

Comment:
Nothing more than for readability and re-usage purposes. Should I just define the whole thing in captureLogs() functions?